.github/workflows/ci.yml

```diff
@@ -1,7 +1,8 @@
 name: Continuous Integration
 
 on:
-  pull_request_target: {}
+  # pull_request_target: {}
+  pull_request: {} # TEST TRIGGER - DO NOT MERGE
```

Comment:
Adding a comment here that will be made outdated by reverting ff47a47c520dfb453124999d6d8968cf3fac309d